docs/config.rst

```diff
@@ -476,7 +476,8 @@ unique_id of the node) and global patterns.
 Rules:
 
  - if multiple node-specific entries reference the same unique_id, only the first will be in effect - all others will be ignored
- - if both the **node** and **interfaces** attributes are specified and a node's unique_id is a match, but the topology information is not, then the overall match will fail and the global patterns will not be considered
+ - if the **node** and **interfaces** or **model** attributes are specified and a node's unique_id is a match, but the topology information or model information are not, then the overall match will fail and the global patterns will not be considered
+ - **model** can't be the only matching attributes, **node** or **interfaces** are mandatory to have a valid match
```

Comment:
Hi,
Sorry for the delay, I was in vacation.

> If I'm understanding this correctly, shouldn't node and model be mutually exclusive? I don't see the case where both are needed. So I'd rephrase this as:

Yes, I think it's a nonsense to use the two together, but it won't result in an error so we thought it was best to explain what would happen.

> * if both the **node** and **interfaces** attributes are specified and a node's unique_id is a match, but the topology information is not, then the overall match will fail and the global patterns will not be considered
> * if both the **model** and **interfaces** attributes are specified and a node's model is a match, but the topology information is not, then the overall match will fail and the global patterns will not be considered
> * **node** and **model** can't be specified in the same pattern

OK for me.

> The model can't be the only matching attr is not fulfilled with the current implementation. What's the reasoning for this rule?

As I remember, it's was mainly because we didn't really needed this use case and this allowed to only operate a very minor modification in the code.